alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetAlertsAction.kt

```diff
@@ -103,39 +107,75 @@ class TransportGetAlertsAction @Inject constructor(
             .sort(sortBuilder)
             .size(tableProp.size)
             .from(tableProp.startIndex)
+        var alertIndex = AlertIndices.ALL_ALERT_INDEX_PATTERN
+        if (!getAlertsRequest.alertIndex.isNullOrEmpty()) {
+            alertIndex = getAlertsRequest.alertIndex
+            getAlerts(searchSourceBuilder, alertIndex, actionListener, user)
+        } else if (getAlertsRequest.monitorId != null) {
+            transportGetMonitorAction.execute(
+                GetMonitorRequest(
+                    getAlertsRequest.monitorId,
+                    -3L,
+                    RestRequest.Method.GET,
+                    FetchSourceContext.FETCH_SOURCE
+                ),
```

Comment:
should we move this to a common util and try to re-use across places wherever we need to fetch the monitor config?

Reply:
let's get this PR in along with #563 and I will do that as a follow up.